Jenkinsfile

```diff
@@ -28,11 +21,31 @@ pipeline {
                 branch 'master'
             }
             steps {
-                sh 'python3 progress.py us --csv >> /var/www/papermar.io/html/reports/progress_us.csv'
-                sh 'python3 progress.py us --shield-json > /var/www/papermar.io/html/reports/progress_us_shield.json'
+                sh 'mkdir reports'
```

Comment:
IDK if you've tested all this already, but shouldn't this be `mkdir -p reports` so the step doesn't fail if `reports/` already exists?

Reply:
It could be that stashed directories are never existing by default, I'm not a Jenkins expert :shipit:

Reply:
The workspace is fresh every time, and if it's not, I'm cool with this being an error (cause that's a bad sign)